src/wallet/rpcwallet.cpp

```diff
@@ -3169,6 +3169,73 @@ UniValue generate(const JSONRPCRequest& request)
     return generateBlocks(coinbase_script, num_generate, max_tries, true);
 }
 
+UniValue rescanblockchain(const JSONRPCRequest& request)
+{
+    CWallet * const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() > 2) {
+        throw std::runtime_error(
+            "rescanblockchain (\"start_height\") (\"stop_height\")\n"
+            "\nRescan the local blockchain for wallet related transactions.\n"
+            "\nArguments:\n"
+            "1. \"start_height\"    (number, optional) block height where the rescan should start\n"
+            "2. \"stop_height\"     (number, optional) block height where the rescan should stop\n"
```

Comment:
nit: add "If omitted, rescan stops at the chain tip."

Reply:
Please specify if the scan stops before or after this block.

I think semantically it makes more sense to provide an inclusive range (if I say "scan blocks 10-20", not scanning 20 seems confusing). 

If you really want an exclusive range, providing the number of blocks to scan (e.g., "scan 10 blocks starting at 10") has better ux, but still worse than inclusive.